.env.dist

```diff
@@ -0,0 +1,4 @@
+AWS_ACCESS_KEY_ID=

```

Comment:
Když nad tím tak přemýšlím, tak ten .env soubor je tam takovej plonkovej, když to nemá `docker-compose.yml`.

Jestli se ti ho tam nechce dodělat, tak bych alespoň hodil do dokumentace, že na spuštění testů je potřeba :
`docker run -it --env-file=$(pwd)/.env -v $(pwd):/code --workdir=/code composer composer ci`

Reply:
fixed